Formula/zim.rb

```diff
@@ -0,0 +1,37 @@
+class Zim < Formula
+  desc "Graphical text editor used to maintain a collection of wiki pages"
+  homepage "http://zim-wiki.org/"
+  url "https://github.com/jaap-karssenberg/zim-desktop-wiki/archive/0.67.tar.gz"
+  sha256 "6817a4eb7be2326870810e4f4bc57c88128b2087752a8bd54953c95357b919fa"
+  head "https://github.com/jaap-karssenberg/zim-desktop-wiki.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "gtk+"
+  depends_on "python"
+  depends_on "pygtk"
+  depends_on "pygobject"
+  depends_on "pygtksourceview" => :optional
+  depends_on "graphviz" => :optional
+
+  resource "pyxdg" do
+    url "https://files.pythonhosted.org/packages/source/p/pyxdg/pyxdg-0.25.tar.gz"
+    sha256 "81e883e0b9517d624e8b0499eb267b82a815c0b7146d5269f364988ae031279d"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec / "lib/python2.7/site-packages"
+    resource("pyxdg").stage do
+      system "python", *Language::Python.setup_install_args(libexec/"vendor")
+    end
+    ENV["XDG_DATA_DIRS"] = libexec/"share"
+    ENV.prepend_create_path "PYTHONPATH", libexec / "lib/python2.7/site-packages"
+    system "python", "./setup.py", "install", "--prefix=#{libexec}", "--skip-xdg-cmd"
+    bin.install Dir[libexec / "bin/*"]
+    bin.env_script_all_files libexec / "bin", :PYTHONPATH => ENV["PYTHONPATH"], :XDG_DATA_DIRS => libexec/"share"
+    pkgshare.install "zim"
+  end
+
+  test do
+    system "#{bin}/zim", "--version"
```

Comment:
Is it possible to have a test that checks more of the functionality?

Reply:
Pretty sure even this launches the GUI, from memory. Maybe I'm misremembering though?

Reply:
@domt4 It doesn't launch the GUI, I'm afraid.
@dunn What's the benchmark for testing that I should be looking for with `brew test`? I sort of figured `--version` would be sufficient (based off of other recipes I've used), but I'm happy to do more :-) I can have it launch the GUI manual, but I think you then have to manually close it.

Reply:
@dafyddcrosby No need to be afraid or whatnot; more than happy if this doesn't throw the GUI up for users :smile:.

Reply:
@dafyddcrosby we try to use `--version` (or `--help` etc.) only when we can't do anything more thorough.  If that's the only command that doesn't open a GUI, then that's what we'll do.

Reply:
I poked around `zim` and confirmed that `zim --version` does not launch the GUI, and it doesn't look like there's much other non-GUI functionality to exercise, so this will be fine, I think. (There's a client-server mode, but it depends on previously-created user data, and doesn't really look like something we want to muck around with in `brew test`, imho.)